docker/3.9/Ubuntu/Dockerfile

```diff
@@ -31,4 +29,4 @@ RUN pip install --no-cache-dir tiatoolbox
 ENV VIRTUAL_ENV=/opt/venv
 ENV PATH="/opt/venv/bin:$PATH"
 
-CMD ["python"]
+# CMD ["python"]
```

Comment:
Why this link is commented?

Reply:
This is commented out to set the entry point to bash script. Otherwise, the docker container can only run Python commands. 
This change will make the functionality of the docker image more versatile.